go/chat/attachment_httpsrv.go

```diff
@@ -350,3 +386,35 @@ func (c *CachingAttachmentFetcher) FetchAttachment(ctx context.Context, w io.Wri
 
 	return nil
 }
+
+func (c *CachingAttachmentFetcher) DeleteAssets(ctx context.Context,
+	convID chat1.ConversationID, assets []chat1.Asset, ri func() chat1.RemoteInterface, signer s3.Signer) (err error) {
+	defer c.Trace(ctx, func() error { return err }, "DeleteAssets")()
+
+	if len(assets) == 0 {
+		return nil
+	}
+
+	// Delete the assets locally
+	for _, asset := range assets {
+		os.Remove(asset.Path)
```

Comment:
This is the remote path, I don't think this is going to work. Is there a test that checks to see if the file is removed?

Reply:
yeah  :(  fixed & test added